docs/trex_release-notes.md

```diff
@@ -13,6 +13,27 @@ See also: [Known Issues]({{site.baseurl}}/docs/trex_known_issues.html)
 
 ----
 
+### Tableau Dashboard Extensions API version 1.6
+
+*September 2021*
+
+* Tableau Dashboard Extensions API library: `tableau.extensions.1.6.0.js` <br>(download or clone the Extensions API repository on [GitHub](https://github.com/tableau/extensions-api){:target="_blank"}.) <br/>
+
+* Download [Tableau Desktop 2021.3](https://www.tableau.com/support/releases){:target="_blank"} or [Tableau Server 2021.3](https://www.tableau.com/support/releases/server){:target="_blank"}.
+
+About this release:
+
+* Tableau Viz is here! <br/>
+
+![Tableau Viz SVG image]({{site.baseurl }}/assets/vizapi_demo3.svg)
+
+Starting with version 1.6 of the Dashboard Extensions API library and Tableau 2021.3, you can now add Tableau visualizations to your dashboard extensions. Tableau Viz takes a declarative description of your visualization and renders it as an SVG file that you can embed in your extension.  Version 1.6 of the Dashboard Extensions library adds the [`tableau.extensions.createVizImageAsync`]({{site.baseurl}}/docs/interfaces/extensions.html#createvizimageasync){:target="_blank"} method, which takes the JSON description of the image as an input.<br/> 
```

Comment:
@johnDance  Good eyes. I'll fix the reference to SVG file here. Thanks!

About the reference to documenting the API outside of the main documentation. I assume that by "main documentation" you are referring to the TypeDoc generated reference pages for the Extensions API. I agree that we should have a links from `vizCreateImageAsync` that go to `trex_tableau_viz` and `trex_tableau_viz_ref`  --That was going to be my next ask. :)  I think it would be too much to put everything about the Tableau Viz grammar/syntax in the TypeScript source file.

Reply:
Okay, please let me know what the link should be.

Reply:
"which takes the JSON description of the image as input", I believe the function createVizImageAsync takes a JS object (and we, the extensions team, convert it to JSON on our end). Should we be more explicit here? JS object vs JSON description?

Reply:
@alicemhsiao - That was my mistake. It should be JS object. I'll fix this.  I was a little fuzzy about what I was calling a JSON object.

Reply:
@alicemhsiao  I rewrote the description of what to do for the `Invalid JSON` error. 

to this: 

The `inputSpec` is a JavaScript object that Tableau converts to JSON for processing. The `inputSpec` needs to be in the correct format and must include all required elements. You must encode columns and fields. For the list of required elements, see [Tableau Viz Reference]({{site.baseurl}}/docs/trex_tableau_viz_ref.html).